app/models/trasaction.js

```diff
@@ -0,0 +1,27 @@
+module.exports = (sequelize, DataTypes) => {
+  const Transaction = sequelize.define(
```

Comment:
do you think trasaction is an expressive name for this model?

Reply:
Is BoughtAlbums better?

Reply:
thats better! also purchases would be a good name